paddle/fluid/framework/device_worker.h

```diff
@@ -101,6 +105,7 @@ class PullDenseWorker {
   float squared_sum_epsilon_ = 1e-4;
   std::mutex mutex_for_mean_scale_;
   float total_batch_num_ = 0;
+  std::map<uint64_t, std::vector<std::vector<float>>> dense_regions_; 
```

Comment:
the name of dense regions is ambiguous, it can be model parameters or gradients. If it is a buffer for memcpy, just give it a meaningful name can be better.